pkg/disk/utils_test.go

```diff
@@ -300,6 +300,58 @@ func (m *MockDisks) AddDisk(t testing.TB, path string, diskID []byte) {
 	}
 }
 
+func TestGetAvailableDiskCountFromAnnotation(t *testing.T) {
+	tests := []struct {
+		name     string
+		node     *corev1.Node
+		expected int
+		wantErr  bool
+	}{
+		{
+			name:     "Empty node",
+			node:     nil,
```

Comment:
I think we don't expect nil node? How about testing for nodes that don't have this annotation?

BTW, test for getAvailableDiskCount ? So we also cover the fallback behavior.